bin/_blackbox_common.sh

```diff
@@ -64,6 +64,15 @@ SECRING="${KEYRINGDIR}/secring.gpg"
 : "${DECRYPT_UMASK:=0022}" ;
 # : ${DECRYPT_UMASK:=o=} ;
 
+# Is this a blackbox repo?
+function is_blackbox_repo() {
```

Comment:
Would this work?
`# Checks if $1 is is null, and if $1/keyrings`
`# is a directory`